docs/Releases.md

```diff
@@ -13,6 +13,307 @@ use a date based versioning system.  Now, a release version can look
 like `17.11` where the "major" number is the year and the "minor" number
 is the month.
 
+## v19.05 (5/19): Shared Core Mode, Major Architectural/API/Initialization/Signal Handling Changes, Advanced Rings Changes, Stats Updates, CI PR Review, LPM Firewall NF, Payload Search NF, TTL Flags, minor improvements and bug fixes.
```

Comment:
Replace `Major Architectural/API/Initialization/Signal Handling Changes` with `Major Architectural Changes`.  Should elaborate on the rest of it in the details.

In general, keep title succinct and expand on it in the bullets.